dashboard/src/components/Connect.tsx

```diff
@@ -63,7 +63,6 @@ export default function Connect({ closable }: { closable: boolean }) {
         }
       }}
       backEnabled={backEnabled}
-      closeable={closable}
```

Comment:
if we just pass `false` here, then there no way back if you click on "connect", right?

Reply:
Yes. If we do that then there's no way to close this dialog:
<img width="641" alt="image" src="https://user-images.githubusercontent.com/5357435/225324901-35452889-4b45-41b1-a8ed-33cdd7bc1d82.png">

Reply:
I'd suggest maybe taking a look at how the `nextEnabled` works and do something similar